docs/en/restful_api.md

```diff
@@ -0,0 +1,19 @@
+# Restful API
+
+## Launch Service
```

Comment:
All restful APIs are supposed to be presented in this guide

Reply:
Added